row_batch.go

```diff
@@ -10,13 +10,15 @@ type RowBatch struct {
 	paginationKeyIndex int
 	table              *TableSchema
 	fingerprints       map[uint64][]byte
+	columns            []string
 }
 
 func NewRowBatch(table *TableSchema, values []RowData, paginationKeyIndex int) *RowBatch {
 	return &RowBatch{
 		values:             values,
 		paginationKeyIndex: paginationKeyIndex,
 		table:              table,
+		columns:            ConvertTableColumnsToStrings(table.Columns),
```

Comment:
Only used in tests.

Reply:
Maybe we should move this function into the test only (as I can see, only in `row_batch_test.go`), to not cause additional confusion? Certainly I'm a bit confused when I looked at this code (and maybe you too given that you left a comment here? 😛 )

Reply:
Alternatively, we can keep this method, and use this method to sanity check that `columns` and `table.Columns` match up?